test/signalrclient-e2e-tests/Build/VS2013/signalrclient-e2e-tests.vcxproj

```diff
@@ -0,0 +1,67 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\..\..\..\packages\cpprestsdk.2.4.0.1\build\native\cpprestsdk.props" Condition="Exists('..\..\..\..\packages\cpprestsdk.2.4.0.1\build\native\cpprestsdk.props')" />
+  <Import Project="..\..\..\..\Build\SignalRClient.Build.Settings" />
+  <PropertyGroup Label="Globals">
+    <ProjectGuid>{6006C96A-29F0-4B18-8DDD-764DC3419E2F}</ProjectGuid>
+    <Keyword>Win32Proj</Keyword>
+    <RootNamespace>signalrclient-e2e-tests</RootNamespace>
+    <ProjectName>signalrclient-e2e-tests</ProjectName>
+    <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\..\..\..\</SolutionDir>
+    <ConfigurationType>Application</ConfigurationType>
+  </PropertyGroup>
+  <Import Project="..\..\..\..\Build\Config.Definitions.props" />
+  <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
+  <Import Project="$(VCTargetsPath)\Microsoft.Cpp.props" />
+  <Import Project="$(UserRootDir)\Microsoft.Cpp.$(Platform).user.props" Condition="exists('$(UserRootDir)\Microsoft.Cpp.$(Platform).user.props')" Label="LocalAppDataPlatform" />
+  <PropertyGroup Label="UserMacros">
+    <NuGetPackageImportStamp>f5caf114</NuGetPackageImportStamp>
+  </PropertyGroup>
+  <ItemDefinitionGroup>
+    <ClCompile>
+      <PreprocessorDefinitions>_CONSOLE;_LIB;NO_SIGNALRCLIENT_EXPORTS;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <AdditionalIncludeDirectories>..\..\..\gtest-1.7.0\include;..\..\..\..\include;..\..\..\..\src\signalrclient;..\..\..\..\include\signalrclient;..\..\..\..\packages\cpprestsdk.2.4.0.1\build\native\include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+    </ClCompile>
```

Comment:
`..\..\..\..\src\signalrclient` this is wrong - these are end to end tests and they should not rely on non-public headers.